drake/solvers/test/testQP.m

```diff
@@ -45,6 +45,29 @@
 testSolvers(prog,solvers,1e-3);
 warning(w);
 
+fprintf('****************************************\n');
+fprintf(' min_x (.5*x(1)^2+x(2)^2+x(1)*x(2) + x(1)+2*x(2)), subj to 5<=x(1)<=10;-20<=x(1)+4*x(2)<10\n');
+fprintf('****************************************\n');
+prog = QuadraticProgram(eye(2),[1;0],[],[],[],[],[],[]);
+prog = prog.addCost(QuadraticConstraint(-inf,inf,[0 1;1 0],zeros(2,1)));
+prog = prog.addCost(LinearConstraint(-inf,inf,[0 2]));
```

Comment:
this fprintf looks wrong, too.  am i misreading?  isn't it just `2*x(1)*x(2) + x(2)`?  maybe with a factor of .5?